test.jsx

```diff
@@ -0,0 +1,80 @@
+import test from 'tape';
+import elementToString from './index';
+import element from 'virtual-element';
+
+test('Simplest form of transform', (t) => {
+  const actual = elementToString(<div></div>);
+  const expected = '<div/>';
+
+  t.equals(actual, expected);
+
+  t.end();
+});
+
+test('Component', (t) => {
+  const Component = {
+    render: function () {
+      return <div></div>;
+    },
+    name: 'Component'
+  };
+
+  const actual = elementToString(<Component />);
+  const expected = '<Component/>';
+
+  t.equals(actual, expected);
+
+  t.end();
+});
+
+test('Component with attributes', (t) => {
+  const Component = {
+    render: function () {
+      return <div></div>;
+    },
+    name: 'Component'
+  };
+
+  function callback () {}
+
+  const actual = elementToString(<Component string='string' callback={callback} boolean={true} />);
+  const expected = '<Component string=\'string\' callback={function} boolean={true}/>';
+
+  t.equals(actual, expected);
+
+  t.end();
+});
+
+test('Nested components', (t) => {
+  const ParentComponent = {
+    render: () => {
+      return <div></div>;
+    },
+    name: 'ParentComponent'
+  };
+
+  const ChildComponent = {
+    render: () => {
+      return <div></div>;
+    },
+    name: 'ChildComponent'
+  };
+
+  const actual = elementToString(<ParentComponent><ChildComponent>child</ChildComponent></ParentComponent>);
+  const expected = '<ParentComponent><ChildComponent>child</ChildComponent></ParentComponent>';
```

Comment:
@ellell what's your thoughts around indentation? This could get pretty hard to read if it's a bigger component. Would be convenient to get indentation done in here automatically...